lib/steps/setupRedirects.js

```diff
@@ -60,7 +60,7 @@ const setupRedirects = (publishPath) => {
   const sortedDynamicRedirects = getSortedRedirects(dynamicRedirects);
 
   // Assemble redirects for each route
-  [...sortedStaticRedirects, ...sortedDynamicRedirects].forEach(
+  [...staticRedirects, ...sortedDynamicRedirects].forEach(
```

Comment:
@lindsaylevine: I was surprised to learn the other day that `.sort()` is destructive (array modified in place), so this change wouldn't have done anything.

> The sort() method sorts the elements of an array *in place* and returns the sorted array.
>
> *From: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort*

If the issue was with the way the redirects are sorted in L55-57, putting `staticRedirects` here wouldn't change anything, since the vars `staticRedirects` and `sortedStaticRedirects` are referencing the same array. To test if the issue is with the sorting in L55-57, maybe try commenting out L59? 🤷 Otherwise, I'm at a loss, too...